src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -85,9 +85,9 @@
     <!-- gather the test archives for upload -->
     <ItemGroup>
       <ForUpload Include="$(TestArchivesRoot)**/*.zip" />
-      <ForUpload Include="$(AnyOSTestArchivesRoot)*.zip" />
```

Comment:
This was because we now have an additional folder that identifies the TestTFM targetted. eg. bin\tests\AnyOS.AnyCPU.Debug\archive\tests\netcoreapp1.0\Microsoft.CSharp.Tests.zip